.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: Feature request
+labels: feature request
+assignees: ahurli, BainanXia, danielolsen, jon-hagg, rouille
+
+---
+
+ # :rocket:
+
+- [ ] Is your feature request essential for your project?
+
+
+### Describe the workflow you want to enable
+A clear and concise description of what can be enhanced, e.g., "I wish I could do [...]"
+
+### Describe your proposed implementation
+This should provide a description of the feature request, e.g.:
+* "The class `Foo` should have a new method `bar` that allows to [...]"
+* "Function `foo` needs a new arguments `bar` to set [...]"
+* "Create a new function `foo` to calculate [...]"
+
+If applicable, try to write a docstring for the desired feature.
```

Comment:
`If applicable, try to write a docstring for the desired feature. If you are not familiar with writing docstrings, you can specify a list of inputs and outputs as necessary, with the types of each (e.g. floating-point number, integer, list, dict-like mapping, data frame [2D table]).`?

Reply:
> `If applicable, try to write a docstring for the desired feature. If you are not familiar with writing docstrings, you can specify a list of inputs and outputs as necessary, with the types of each (e.g. floating-point number, integer, list, dict-like mapping, data frame [2D table]).`?

Would that be harder than writing doc strings assuming doc strings referring to the description of what a function do not including the I/O?

Reply:
I guess for us, a docstring is two parts: a very short user-facing description of what the function does, plus a detailed descriptions of the inputs and outputs. Ideally, we want to encourage the requester to give us both, even if they have never heard of a docstring before.

Reply:
Agree. Maybe we can be more explicit with what we mean by doc strings here:

- A description of what the function does
- I/O variable definitions 

And ask the user to provide as much info as they could come up with.

Reply:
I have elaborated on docstring in 468297f8e35e206a979225ab361e363cd55bcb7f